packages/osd-optimizer/src/worker/webpack.config.ts

```diff
@@ -235,6 +235,18 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
             },
           },
         },
+        {
+          test: /\.js$/,
+          include: /node_modules[\\/]@?reactflow/,
```

Comment:
Cant we add this to the section above since the same exception applies?

Reply:
done